inlong-dataproxy/dataproxy-source/src/test/resources/common.properties

```diff
@@ -20,6 +20,8 @@ metricDomains=DataProxy
 metricDomains.DataProxy.domainListeners=org.apache.inlong.dataproxy.metrics.prometheus.PrometheusMetricListener
 metricDomains.DataProxy.snapshotInterval=60000
 
+node.startup.via.meta.config.file.enable=true
```

Comment:
Please add more description for this param.
otherwise, could we use `startup.using.local.meta.file.enable` to make this param easy to understand?

Reply:
It's really long, I was thinking about it when I first named it :)